wiki/People/Beatmap_Nominators/en.md

```diff
@@ -48,6 +49,13 @@ Modders that aim to apply to the Beatmap Nominators must fulfill the following c
   + osu!: 1100
 + It is usually a good idea to keep their score well above these numbers as they can vary slightly depending on the amount of applicants.
 
+### How the new potential Beatmap Nominators are chosen
++ A doublecheck of the `Mod Score` and the `Kudosu Score` will be done to all applications, in order to find out if they reached the `Required Minimum Score`.
```

Comment:
"The `Mod Score` and  `Kudosu Score` of all applicants will be double checked to confirm if they"
less wordy and fewer unused words